doc/index.md

```diff
@@ -1,16 +1,16 @@
 # Building a WildFly server for the cloud
 
 In order to provision a WildFly server for the cloud, you have first to integrate the [WildFly Maven plugin](https://github.com/wildfly/wildfly-maven-plugin/)
-into the pom.xml file of your project.
+into the `pom.xml` file of your project.
 
-The "package" goal of the plugin allows you to:
+The `package` goal of the plugin allows you to:
 
 * Provision a WildFly server tailored to your needs.
-* Incorporate custom content (keystores, properties files, ...).
+* Incorporate custom content (keystores, properties files, etc).
 * Execute CLI scripts.
 * Deploy your application.
 
-The generated server is then ready to be installed inside a Docker Image to be deployed in your cluster.
```

Comment:
@rhusar would it make sense to use "container image" instead of "Docker/Podman/Whatever-is-the-next-tool-that-builds-it image"  ? ;)

Reply:
(I'm asking here but we should use the same terminology in the wildlfy-maven-plugin too)